patzue.py

```diff
@@ -8,83 +7,126 @@
 import os
 import io
 
-##Directorio de trabajo
-dir = os.chdir("C:/Users/ALIENWARE/Documents/censo2020/mich")
-
-

```

Comment:
No es tan buena práctica publicar directorios locales.

Reply:
En el stream pasado de @haro_ca_ en su vídeo https://youtu.be/S-gzW1zf2P4?t=166 es buena práctica usar los path's relativos.